src/main/java/com/buddy/tutorial/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.buddy.tutorial.controller;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.buddy.tutorial.model.User;
+import com.buddy.tutorial.service.UserService;
+
+@RestController
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserController {
+
+    private UserService userService;
```

Comment:
when performing constructor injection, use final. If it's a field level injection, you can keep as it is